test/e2e/exec_test.go

```diff
@@ -286,7 +286,7 @@ var _ = Describe("Podman exec", func() {
 	It("podman exec preserves container groups with --user and --group-add", func() {
 		SkipIfRemote("FIXME: This is broken SECCOMP Failues?")
 
-		dockerfile := `FROM fedora-minimal
+		dockerfile := `FROM registry.fedoraproject.org/fedora-minimal
```

Comment:
I've run into troubles with this repo in the past, will keep my fingers crossed.

Reply:
This is (hopefully) only temporary until Brent can get local caching working.